pkg/cmd/template/data_values_flags.go

```diff
@@ -288,11 +289,18 @@ func (DataValuesFlags) libraryRefAndKey(key string) (string, string, error) {
 		if len(keyPieces[0]) == 0 {
 			return "", "", fmt.Errorf("Expected library ref to not be empty")
 		}
-		return keyPieces[0], keyPieces[1], nil
-
-	default:
-		return "", "", fmt.Errorf("Expected at most one library-key separator '%s' in '%s'", libraryKeySep, key)
+		if strings.HasPrefix(keyPieces[0], ref.LibrarySep) {
+			return keyPieces[0], keyPieces[1], nil
+		}
+		// the LibrarySep is part of the path
+		return "", key, nil
+	case 3:
+		if strings.HasPrefix(keyPieces[0], ref.LibrarySep) {
+			// the second LibrarySep is part of the path
+			return keyPieces[0], keyPieces[1] + libraryKeySep + keyPieces[2], nil
+		}
 	}
+	return "", "", fmt.Errorf("Expected at most one library-key separator '%s' in '%s'", libraryKeySep, key)
```

Comment:
This seems to contradict the fact that you have `case 3` in your switch statement. Currently, this code reads to me as:
- It's okay to have two ":"s if the whole string is prefixed with "@". Everything after the first ":" is the path to the file.
- otherwise, two or more ":"s is an error.

On a Mac, there would certainly be an error later on about trying to read the file with a weird name, but I wanted to point out that the bullets presented in this comment did not match my assumption that multiple ":"s would only be considered in the windows case.

Reply:
I updated the PR to allow 0+ `:` characters in any file path. I also learned that file names in unix can contain `:` so we should also allow them.